src/pages/_app.tsx

```diff
@@ -32,13 +34,15 @@ export default function App({ Component, pageProps }: AppProps) {
       </Head>
       <main>
         <ClientOnly>
-          <ResumeFormProvider>
-            <CvContextProvider>
-              <MainLayout>
-                <Component {...pageProps} />
-              </MainLayout>
-            </CvContextProvider>
-          </ResumeFormProvider>
+          <ConfigProvider locale={enUs}>
+            <ResumeFormProvider>
+              <CvContextProvider>
```

Comment:
we should combine ResumeFormProvider and ResumeFormProvider into one

Reply:
I believe we should

Reply:
We use WizardContext as a default for CVContext